Hazel/Dtls/DtlsConnectionListener.cs

```diff
@@ -1331,8 +1341,10 @@ protected override void QueueRawData(ByteSpan span, IPEndPoint remoteEndPoint)
                         ref outgoingRecord
                     );
 
-                    base.QueueRawData(packet, remoteEndPoint);
+                    queuedSpan.Recycle();
```

Comment:
I don't think I see the associated AddUsage() call for this one - I'd assume it should be the 2nd line of the for block?

Reply:
Ahhh I think I might see how this works - this queue is only accessed by this same method so you can ensure the same number of AddUsage()/Recycle() pairs

Reply:
Refactored this away by with the using pattern